packages/paste-design-tokens/tokens/aliases/color-palette.yml

```diff
@@ -5,12 +5,24 @@ aliases:
   palette-blue-30: "#dceaf4"
   palette-blue-40: "#b3d3e9"
   palette-blue-50: "#56a4d9"
-  palette-blue-60: "#0075c3"
+  palette-blue-60: "#027ac5"
   palette-blue-70: "#005ea6"
   palette-blue-80: "#003e82"
-  palette-blue-90: "#001f57"
+  palette-blue-90: "#032c5e"
   palette-blue-100: "#00092b"
 
+  ## Blues transparent based on primary blue
+  palette-blue-60-transparent-10: "rgba(2, 123, 200, 0.1)"
+  palette-blue-60-transparent-20: "rgba(2, 123, 200, 0.2)"
+  palette-blue-60-transparent-30: "rgba(2, 123, 200, 0.3)"
+  palette-blue-60-transparent-40: "rgba(2, 123, 200, 0.4)"
+  palette-blue-60-transparent-50: "rgba(2, 123, 200, 0.5)"
+  palette-blue-60-transparent-60: "rgba(2, 123, 200, 0.6)"
+  palette-blue-60-transparent-70: "rgba(2, 123, 200, 0.7)"
+  palette-blue-60-transparent-80: "rgba(2, 123, 200, 0.8)"
+  palette-blue-60-transparent-90: "rgba(2, 123, 200, 0.9)"
+  palette-blue-60-transparent-100: "rgba(2, 123, 200, 1)"
```

Comment:
With the `blue-60` change, the rgb value needs to change to 2, 122, 197

Reply:
Good catch!